mmcv/runner/hooks/lr_updater.py

```diff
@@ -212,5 +213,70 @@ def get_lr(self, runner, base_lr):
         else:
             progress = runner.iter
             max_progress = runner.max_iters
-        return self.target_lr + 0.5 * (base_lr - self.target_lr) * \
-            (1 + cos(pi * (progress / max_progress)))
+        if self.as_ratio:
+            target_lr = base_lr * self.target_lr
+            return target_lr + 0.5 * (base_lr - target_lr) * \
+                (1 + cos(pi * (progress / max_progress)))
+        else:
+            return self.target_lr + 0.5 * (base_lr - self.target_lr) * \
+                (1 + cos(pi * (progress / max_progress)))
```

Comment:
It can merge into one statement.
```
        if self.as_ratio:
            target_lr = base_lr * self.target_lr
        return target_lr + 0.5 * (base_lr - target_lr) * \
               (1 + cos(pi * (progress / max_progress)))
```